SGoopas/Assets/Scripts/Game/3D/PlayerCamera.cs

```diff
@@ -51,4 +53,17 @@ public class PlayerCamera : MonoBehaviour {
     public void SwitchTo3D(Cancellable cancellable) {
         cancellable.PerformCancellable(Follow3DPlayer, Follow2DPlayer);
     }
+
+    public void FocusOnObject(GameObject focusObject) {

```

Comment:
Maybe let the caller pass in the distance from target? This method would be great for any other scripted events (although idk how many more we'll have at this point)

Reply:
yee I considered that. I'm actually gonna do it, I think it's a good idea.